mix.exs

```diff
@@ -42,20 +42,19 @@ defmodule Delta.MixProject do
 
   defp deps do
     [
-      {:gen_stage, "~> 1.0"},
-      {:hackney, "~> 1.15"},
-      {:cowlib, "~> 2.8"},
+      {:gen_stage, github: "paulswartz/gen_stage", branch: "consumer-supervisor-spec"},
```

Comment:
Can we add a comment here indicating when we can switch back to the official package? Maybe just a link to your PR would be fine, or a link to the commit it introduced to `master`. (also, nice, José-approved!)